Doc/whatsnew/3.13.rst

```diff
@@ -189,6 +189,14 @@ typing
   check whether a class is a :class:`typing.Protocol`. (Contributed by Jelle Zijlstra in
   :gh:`104873`.)
 
+
+pdb
+---
+
+* Add ability to move between chained exceptions during post mortem debugging in :func:`~pdb.pm` using
+  the new ``exceptions [exc_number]`` command for Pdb. (Contributed by Matthias
+  Bussonnier in :gh:`106676`.)
```

Comment:
Sorry one last nit. The modules here are sorted alphabetically, so this needs to be between path lib and sqlite.

Reply:
No problem, done and rebased/squashed a bit for a cleaner history.

Reply:
> No problem, done and rebased/squashed a bit for a cleaner history.

Please don't use rebases or squashes on CPython PRs, when you can help it :) It interacts badly with the GitHub UI, making it hard for reviewers to see what changed between commits. Everything in CPython is squash-merged anyway, so we really don't care about messy commit history within PRs.

Reply:
Thanks. 

In the future don't squash. We squat on merge anyway, so the commit history doesn't end up in the commit log. But it's useful to have it on the PR, and it's easier to review just the last commit and see what you've changed.

Reply:
Ok, I can also un-squash as well I have the reflog.